File: java-main/java-main/2D-Array/p8.java
import java.util.*;
class Demo{
	public static void main(String[]args){

		Scanner sc=new Scanner(System.in);

		System.out.println("Enter number of planes");
		int plane=sc.nextInt();
		System.out.println("Enter number of rows");
		int rows=sc.nextInt();
		System.out.println("Enter number of cloumns");
		int col=sc.nextInt();
		
		int arr[][][] = new int[plane][rows][col];

		for(int i=0;i<plane;i++){
			for(int j=0;j<rows;j++){
				for(int k=0;k<col;k++){
					arr[i][j][k]=sc.nextInt();
				}
			}
		}
		for(int[][] x : arr){
			for(int[] y : x){
				for(int z:y){

				System.out.print(z + " ");
				}
				System.out.println();
			}
			System.out.println();
			
		}
	}

}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/session4/p8.c
/* 
   d d d d
   c c c 
   b b
   a
   */


#include<stdio.h>
void main(){

        int rows;

        printf("Enter no of rows:\n");
        scanf("%d",&rows);
        

        for(int i=1;i<=rows;i++){
		char ch=96+rows;


                for(int j=rows;j>=i;j--){
			if(i%2!=0){
                        printf("%c ",96+ch);
			}else{
				printf("%c ",64+ch);
			}
			ch--;
		}
		printf("\n");
	
	}

}

